File: hooks/tests/fixtures/codenav/perl/navigation.pl
package BaseReads;
sub base_count { return 1; }
package Reads;
use strict;
use warnings;
use Exporter 'import';
use parent 'BaseReads';
require 'helpers.pl';
sub normalize {
    my ($value) = @_;
    return helper($value);
}
sub run { return normalize(1); }
# sub phantom { return 2; }
1;

File: hooks/tests/fixtures/codenav/r/navigation.R
library(methods)
require(stats)
source("helpers.R")
normalize <- function(
    x, scale = 2) {
    helper(x) / scale
}
apply_normalize = function(x) normalize(x)
ReadSet <- setClass("ReadSet", contains = "BaseSet")
Runner <- R6::R6Class("Runner", inherit = BaseRunner,
    public = list(run = function(x) normalize(x)))
# phantom <- function(x) helper(x)
text <- "phantom <- function(x) helper(x)"
lapply(1:3, FUN = function(x) x)
